src/components/AddTeamLeaderModal.tsx

```diff
@@ -310,7 +310,7 @@ const AddTeamLeaderModal: React.FC<AddLearnerModalProps> = ({
         open={open}
         onClose={onClose}
         showFooter={false}
-        modalTitle={t("TEAM_LEADERS.NEW_TEAM_LEADER")}
+        modalTitle={isEditModal? t("TEAM_LEADERS.NEW_TEAM_LEADER"): t("TEAM_LEADERS.EDIT_TEAM_LEADERS")}
```

Comment:
@AkshataKatwal16 it should be EDIT_TEAM_LEADER

Reply:
updated